integration/specs/InternalOverlay/internalOverlay-1.spec.js

```diff
@@ -14,6 +15,20 @@ describe('InternalOverlay with default position resolver', () => {
         component.waitForExist();
     });
 
+    it('should set focus on trigger when click it', () => {
+        const button = new PageInternalOverlayButton(BUTTON);
+        button.click();
+        expect(button.hasFocusTrigger()).toBe(true);
+    });
```

Comment:
we should remove this test since this is not behavior for the InternalOverlay, since we don't set the focus for anything inside the component this is the default browser behavior and then not make sense test it